vncviewer/DesktopWindow.cxx

```diff
@@ -921,7 +921,16 @@ void DesktopWindow::fullscreen_on()
 
   if (not selectedMonitors and not allMonitors) {
     int n = Fl::screen_num(x(), y(), w(), h());
+#ifdef __APPLE__
+    // This is a workaround for a bug in FLTK, see: https://github.com/fltk/fltk/pull/277
+    int savedLevel;
+    savedLevel = cocoa_get_level(this);
+#endif
     fullscreen_screens(n, n, n, n);
```

Comment:
Can't we pop this out one level to avoid the duplication? It's easy to miss this in future changes.

Reply:
Sure, good idea :+1: